indico/modules/rb_new/blueprint.py

```diff
@@ -91,3 +92,14 @@
 _bp.add_url_rule('/api/admin/attributes', 'admin_attributes', admin.RHAttributes, methods=('GET', 'POST'))
 _bp.add_url_rule('/api/admin/attributes/<int:attribute_id>', 'admin_attributes', admin.RHAttributes,
                  methods=('GET', 'DELETE', 'PATCH'))
+_bp.add_url_rule('/api/admin/rooms/<int:room_id>', 'admin_room', admin.RHRoom)
+_bp.add_url_rule('/api/admin/rooms/<int:room_id>', 'admin_update_room', admin.RHRoomUpdate, methods=('PATCH',))
+_bp.add_url_rule('/api/admin/rooms/<int:room_id>/equipment', 'admin_room_equipment', admin.RHRoomEquipment)
+_bp.add_url_rule('/api/admin/rooms/<int:room_id>/equipment', 'admin_update_room_equipment', admin.RHRoomEquipmentUpdate,
```

Comment:
`RHRoomEquipmentUpdate` -> `RHUpdateRoomEquipment` to be consistent with e.g. `RHDeleteBooking`